src/kernels/gpu_reference_kernel/fp8_kern_types.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
```

Comment:
Please check all new files for copyright notice.

Reply:
Done